doc/api/esm.md

```diff
@@ -338,15 +338,15 @@ command flag enabled.
 * `specifier` {string} The module specifier to resolve relative to `parent`.
 * `parent` {string|URL} The absolute parent module URL to resolve from. If none
   is specified, the value of `import.meta.url` is used as the default.
-* Returns: {Promise}
+* Returns: {string}
```

Comment:
The docs here should include a clear indication that this operation will include sync filesystem operations and therefore will have a performance cost.

Reply:
done: 640c91c